### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by nicklathe**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+labels: problem/bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**I tried this**
+Steps the caused the bug/error.
```
Comment (ncronquist) [inline .github/ISSUE_TEMPLATE/bug_report.md]: the -> that

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+**I expected this**
+A clear and concise description of what you expected to happen.
+
+**Instead this happened**
+A clear and concise description of what the unexpected outcome was.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (ncronquist) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Maybe change the example OS to Windows or Mac OS.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ - Kubectl Version: [e.g. 22]
```
Comment (ncronquist) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I wonder if we should just ask for the output of `kubectl version` here.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (nicklathe) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Good catch.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (nicklathe) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah, I agree. I changed it to `e.g. Windows or Mac OS version`. Are we happy with that wording?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (nicklathe) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yeah, I agree. I changed this to [output from `kubectl version`]

**Comment by dragan** [comment]
Probably should update the CONTRIBUTING doc as part of this PR since we have templates now.

**Review by dragan — CHANGES_REQUESTED** [review]
LGTM! Squash please. I'll update the feature request template as part of my RFC work in #12.

**Review by dragan — APPROVED** [review]
:shipit:

**Pull request merged**